requirements.txt

```diff
@@ -1,8 +1,8 @@
 -i https://pypi.org/simple
 calmjs-parse==1.2.5
-certifi==2020.6.20
+certifi>=2020.6.20
 chardet==3.0.4
 idna==2.10
```

Comment:
Please remove these requirements. They are not used directly by this library and `requests` already depend on them.
```suggestion
```